functions/helpers.js

```diff
@@ -1,3 +1,13 @@
+let options = { 
+    method: 'POST',
+    url: 'https://api.sendinblue.com/v3/emailCampaigns/1/sendTest',
```

Comment:
Don't you want to use https://github.com/sendinblue/APIv3-nodejs-library?

Reply:
done